src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1884,10 +953,10 @@ bool entitiesQuery
   const std::string&               sortOrderList,
   bool                             includeCreDate,
   bool                             includeModDate,
-  const std::string&               apiVersion
+  const std::string&               apiVersion,
+  StringFilter*                    stringFilterP
```

Comment:
As far as I understand, now the string filters are provides as Scopes (within ghte `res` parameter). Thus, probably this is a leftover of an previous version along the history of this PR.